### ISSUE TITLE
`application-duality` false alarm

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#268]
This is the code:

```
[] > foo
  a > @
    "first".as-bytes:0
    "second":1
```

The `application-duality` complains, but it shouldn't.

**Comment by h1alexbel** [comment]
@yegor256 I will take this

**Comment by yegor256** [comment]
@h1alexbel try this one:

```
[] > vertical-bound-method
  eq. > @
    if.
      true
      "second":1
      "first"
      .as-bytes:0
    "first"
```

**Comment by h1alexbel** [comment]
@yegor256 currently it passes, is it expected right?

**Comment by h1alexbel** [comment]
@yegor256 too me it looks invalid, since `if` has three objects: two of them have `@as` (`second`, `first`), while `true` does not have `@as`, so that's why it should be failed. WDYT?

**Comment by yegor256** [comment]
@h1alexbel this is the code I have in `runtime-tests.eo`:

```
[] > vertical-bound-method
  eq. > @
    if.
      true
      "second":1
      "first"
      .as-bytes:0
    "first"
```

This is what prints:

```
[ERROR] org.eolang.runtime-tests:357 The anonymous object 
has duality in the application, which is prohibited (application-duality)
```

**Comment by yegor256** [comment]
@h1alexbel this code is valid, `if` has only two arguments. This code is equivalent to:

```
[] > vertical-bound-method
  eq. > @
    true.if
      "second":1
      "first".as-bytes:0
    "first"
```

**Comment by h1alexbel** [comment]
@yegor256 the issue should gone in `0.0.35`, try again, please

**Comment by yegor256** [comment]
@h1alexbel thanks!
